samples/subsys/mgmt/mcumgr/smp_svr/sample.yaml

```diff
@@ -92,6 +92,7 @@ tests:
       - rd_rw612_bga
       - nrf52840dk/nrf52840
       - nrf54h20dk/nrf54h20/cpuapp/iron
+      - nrf54h20dk/nrf54h20/cpuapp/iron/mcuboot/b0
```

Comment:
move this to sdk-nrf as we do for other tests/samples - see https://github.com/nrfconnect/sdk-nrf/tree/main/samples/zephyr and https://github.com/nrfconnect/sdk-nrf/tree/main/tests/zephyr for examples